README.md

```diff
@@ -1,679 +1,891 @@
 # DataRobot Provider for Apache Airflow
 
-This package provides operators, sensors, and a hook that integrates [DataRobot](https://www.datarobot.com) into Apache Airflow.
-Using these components you should be able to build the essential DataRobot pipeline - create a project, train models, deploy a model,
-score predictions against the model deployment.
+This package provides operators, sensors, and a hook to integrate [DataRobot](https://www.datarobot.com) into Apache Airflow.
+Using these components, you should be able to build the essential DataRobot pipeline - create a project, train models, deploy a model,
+and score predictions against the model deployment.
 
+## Install the Airflow Provider
```

Comment:
Is "Provider" part of the product name or should it be lowercase?

Reply:
I think if it were DataRobot Airflow Provider that could be the full name, but in this case I think you're right and it should be lowercase.